src/Engine/Component/Mesh.cpp

```diff
@@ -7,7 +7,6 @@
 using namespace Component;
 
 Mesh::Mesh() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Member variable 'Mesh::isSelected' is not initialized in the constructor. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AuninitMemberVar)